### PR TITLE
Chassisd do an explicit stop of the config_manager

### PR DESCRIPTION
**Pull request by judyjoseph**
#### Description
Do an explicit task stop of config_manager task in the chassisd 
 
#### Motivation and Context
Fixes issue https://github.com/sonic-net/sonic-buildimage/issues/11955

#### How Has This Been Tested?
Updated the test_chassisd.py as per https://github.com/sonic-net/sonic-mgmt/pull/7150 and made sure it is a consistent PASS

### CODE REVIEW

**Review by prgeor — DISMISSED** [review]

**Comment by prgeor** [comment]
@judyjoseph code coverage

**Comment by shivuv** [comment]
@abdosi : Can you please request for double commit of this into 202205 branch?

File: sonic-chassisd/tests/mocked_libs/sonic_platform/chassis.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"""
+    Mock implementation of sonic_platform package for unit testing
+"""
+
+# TODO: Clean this up once we no longer need to support Python 2
```
Comment (prgeor) [inline sonic-chassisd/tests/mocked_libs/sonic_platform/chassis.py:5]: can we remove this TODO. we no longer support python2

**Review by prgeor — COMMENTED** [review]
@judyjoseph approving with comments. please check

**Review by prgeor — DISMISSED** [review]

**Review by prgeor — APPROVED** [review]

**Comment by amulyan7** [comment]
@judyjoseph can we get this committed to 202205 branch as well?

**Comment by yxieca** [comment]
This change is reverted from 202205 due to build failure. Please ping me when the build failure fix is in.

**Comment by shivuv** [comment]
> @judyjoseph can we get this committed to 202205 branch as well?

@abdosi : Please check it

**Comment by judyjoseph** [comment]
> > @judyjoseph can we get this committed to 202205 branch as well?
> 
> @abdosi : Please check it


> > @judyjoseph can we get this committed to 202205 branch as well?
> 
> @abdosi : Please check it

Raised PR : https://github.com/sonic-net/sonic-platform-daemons/pull/336

**Pull request merged**